core/src/main/scala/query/interpolator/Module.scala

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2017 47 Degrees, LLC. <http://www.47deg.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package freestyle.cassandra
+package query.interpolator
+
+import com.datastax.driver.core.ResultSet
+import freestyle._
+import freestyle.cassandra.api.{SessionAPI, StatementAPI}
+import freestyle.cassandra.query.model.SerializableValueBy
+
+@module
+trait Module {
```

Comment:
I'm not good with naming, but `Module` is not good here, IMHO.

What about `InterpolatorRunner` / `IRunner` / ...?? or something similar?